### DISCUSSION TITLE
Extensions to be able to add to the Lightning screens

### DISCUSSION THREAD

**Discussion by instabot42** [btcpayserver/btcpayserver#3844] [Ideas / Feature Requests]
I'm working on an extension that will add features to allow users to add inbound and outbound liquidity to the Lightning node connected to their store, but it is unclear where to use the UI for this feature.

I'd like to place it on the Lightning page (see screenshot) as either a new section on the main page, or a new tab with a page for the controls for the feature, or maybe add something to the settings page (though this does not seem like a suitable location as this would not be a setting)

This may need changes to the extensions system (still a bit new to this so not sure if this is already supported) and maybe some UX decisions as to where such things should be added.

![BTC_Lightning](https://user-images.githubusercontent.com/35345644/172813222-7d7a6f19-094c-4cc5-a502-8de235b743d9.png)

The existing UI for our feature can be seen at https://synonym.to/blocktank/ for reference, though it is out intention to build a much more BTCPay Server-centric version of this, with a focus on providing inbound liquidity to retailers.

Our product is fully open source with the UI currently a react app (github for the UI can be found at https://github.com/synonymdev/blocktank-ui)

**Comment by bolatovumar** [comment]
Hi @instabot42,

BTCPay has a "Services" section for Lightning that can contain a link to your extension.

![Capture](https://user-images.githubusercontent.com/1934678/176598561-2418f36e-1a47-488b-bc67-77a6d85bf247.PNG)
